### PR TITLE
SWC-6824

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Dependent on https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/930/files

File: src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerImpl.java [hunk 1]
```diff
@@ -626,6 +626,7 @@ private void reconfigureActions() {
     configureChangeStorageLocation();
     configureCreateOrUpdateDoi();
     configureEditProjectMetadataAction();
+    configureProjectHelpAction();
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerImpl.java:629]: New Action

File: src/main/java/org/sagebionetworks/web/client/widget/entity/menu/v3/DefaultEntityActionMenuLayoutUtil.java [hunk 2]
```diff
@@ -160,6 +160,11 @@ public static EntityActionMenuLayout getLayout(EntityType entityType) {
         layout.setPrimaryMenuText(
           EntityTypeUtils.getDisplayName(entityType) + TOOLS_SUFFIX
         );
+        layout.setButtonActions(
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/widget/entity/menu/v3/DefaultEntityActionMenuLayoutUtil.java:163]: Added as a Button Action in the Project action menu layout

File: src/main/webapp/sass/_core.scss [hunk 3]
```diff
@@ -227,7 +227,8 @@ a:focus {
       }
       button.toolsMenuButton,
       .dropdown.open,
-      .MuiButton-root {
+      .MuiButton-root,
+      .MuiIconButton-root {
```
Comment (jay-hodgson) [inline src/main/webapp/sass/_core.scss:231]: Also include MuiIconButtons (in addition to MuiButtons) when styling buttons under the projectMetadata class.  This makes the icon color white

**Review by nickgros — APPROVED** [review]

**Comment by jay-hodgson** [comment]
Dependent on https://github.com/Sage-Bionetworks/SynapseWebClient/pull/5380

**Pull request merged**